src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -1060,15 +1050,14 @@ def readCondorLog(self, job):
             cres = condor.read_events(logfileobj, 1)
             ulog = list(cres)
```

Comment:
We could make it safer like
```
lastEvent= None
for event in logEvents:
   lastEvent = event
```
though from a quick test, it seems to be ~60% slower (for a sample of 10M integers) than the code in the PR.